configs/bs/cluster/disks.yaml

```diff
@@ -0,0 +1,21 @@
+global:
+  format_percent: 90
+  host:
+    - curve-1
+    - curve-2
+    - curve-3
+
+disk:
+  - device: /dev/sdb1
+    mount: /data/chunkserver0
+  - device: /dev/sdc1
+    mount: /data/chunkserver1
+  - device: /dev/sdd1
+    mount: /data/chunkserver2
+    host_exclude:  # for the use case that all hosts have this disk device except some hosts
```

Comment:
We should keep the config template and the implemetion consistent.

Reply:
You mean the comments of `disks.yaml`?

Reply:
`disks.yaml` template and the implementation is consistent with double check. It would be helpful with more specific comments.

Reply:
Done.

Reply:
> We should keep the config template and the implemetion consistent.

Note, disks configure item names `disk` and `host` are used in implementation, not `disks` and `hosts`.
```
global:
  format_percent: 95
  host:
    - curve-1
    - curve-2
    - curve-3
   
  disk:
    - device: /dev/sdb1
      mount: /data/chunkserver0
    - device: /dev/sdc1
      mount: /data/chunkserver1
      format_percent: 90
    - device: /dev/sdd1
      mount: /data/chunkserver2
      exclude:
        - curve-1
    - device: /dev/sde1
      mount: /data/chunkserver3
      host:
        - curve-1
        - curve-2

Reply:
```
Disks struct {
		Global map[string]interface{}   `mapstructure:"global"`
		Disk   []map[string]interface{} `mapstructure:"disk"`
	}